x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -102,23 +99,22 @@ export const entityTypesRt = new t.Type<string[], string, unknown>(
   (arr) => arr.join()
 );
 
-export interface Entity {
-  [ENTITY_LAST_SEEN]: string;
-  [ENTITY_ID]: string;
-  [ENTITY_TYPE]: string;
-  [ENTITY_DISPLAY_NAME]: string;
-  [ENTITY_DEFINITION_ID]: string;
-  [ENTITY_IDENTITY_FIELDS]: string | string[];
-  alertsCount?: number;
-  [key: string]: any;
-}
-
 export type EntityGroup = {
   count: number;
 } & {
   [key: string]: any;
 };
 
-export type InventoryEntityLatest = z.infer<typeof entityLatestSchema> & {
+export type EntityLatest = z.infer<typeof entityLatestSchema>;
+
+export type InventoryEntityLatest = {
```

Comment:
We can't remove `EntityLatest` naw because `getIdentityFieldsValue` and `asKqlFilter` functions from Entity Client accepts `entityLatestSchema`. When we move queries to entity client, we can make the changes to remove  `EntityLatest`